api/meter.go

```diff
@@ -58,46 +60,46 @@ type Latency struct {
 
 // TODO: refactor common fields
 type AppRelaysResponse struct {
-	Count       RelayCounts
-	From        time.Time
-	To          time.Time
-	Application string
+	Count     RelayCounts              `json:"Count"`
```

Comment:
JSON tags explicitly set to avoid disruption to other repos that have not had their struct field names updated yet.